rpm/package.sh

```diff
@@ -79,25 +79,33 @@ function download_flume(){
 
 function copy_cygnus_startup_script(){
     _logStage "######## Copying the cygnus startup script into the apache-flume... ########"
-    rm ${RPM_PRODUCT_SOURCE_DIR}/bin/flume-ng
-    cp $BASE_DIR/target/classes/cygnus-flume-ng ${RPM_PRODUCT_SOURCE_DIR}/bin
+    rm -rf ${RPM_PRODUCT_SOURCE_DIR}/bin/cygnus-flume-ng
```

Comment:
Why is `cygnus-flume-ng` script removed? I think it was correct to remove `flume-ng` script, since it is the native one. `cygnus-flume-ng` replaces it.